salt/repo/online.sls

```diff
@@ -0,0 +1,36 @@
+{%- from "repo/map.jinja" import repo with context %}
+
+{%- if grains["os_family"] == "RedHat" %}
```

Comment:
We only support this environment for now, so let's not add `if`/`then` constructions prematurely.

Reply:
Ok